packages/decompress/decompress.0.8.1/opam

```diff
@@ -20,7 +20,7 @@ build: [
 ]
 
 depends: [
-  "ocaml"      {>= "4.03.0"}
+  "ocaml"      {>= "4.03.0" & < "4.08.0"}
```

Comment:
Why constrain this package? According to `check.ocamllabs.io` it works perfectly?

Reply:
Weird. It uses `Bigarray.Array1.flat_map` that has been removed by ocaml 4.08.0. I found the error in the CI, I am surprised that it works there. Either must be wrong for some reasons

Reply:
The failiures are: https://ci.ocaml.org/log/saved/docker-run-2c51b163a4cfe9f3204aa0475c869367/7c6d68e13c671ec5009a1b390f71bc28794892ec and https://ci.ocaml.org/log/saved/docker-run-f49e35eaf0ef8d303dc20da208511d96/7bae61533195f65825c80c887fc993f179ca4fac

Reply:
As described in the logs it only fails during tests.